package 数组异常;

import java.util.Scanner;

public class testAnArrayOfException {

	@SuppressWarnings("resource")
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		Scanner input = new Scanner(System.in);
		int[]num = new int[5];
		for(int i=0;i<num.length;i++){
			try{
			num[i]=input.nextInt();
			System.out.print(num[i]);
			}
			catch(ArrayIndexOutOfBoundsException e){
				System.out.println("下标异常");
			}
			finally{
				System.out.println(i);
			}
		}
		
			
			
		
	}
}


